src/interaction/InteractionManager.js

```diff
@@ -414,7 +412,7 @@ InteractionManager.prototype.onMouseDown = function (event)
 {
     this.mouse.originalEvent = event;
     this.eventData.data = this.mouse;
-    this.eventData.stopped = false;
+    this.eventData.stopped = event.defaultPrevented;
```

Comment:
I think that is super hacky, instead of trying to handle a stopped event we should just not use the capturing phase.

I feel like doing that and solving problems we have there will be the best solution. I'm looking to try and rewrite the Interaction Manager in the coming weeks. Stay tuned.